packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -90,28 +87,11 @@ export class MonacoEditorProvider {
         @inject(EditorPreferences) protected readonly editorPreferences: EditorPreferences,
         @inject(MonacoQuickOpenService) protected readonly quickOpenService: MonacoQuickOpenService,
         @inject(MonacoDiffNavigatorFactory) protected readonly diffNavigatorFactory: MonacoDiffNavigatorFactory,
-        @inject(ApplicationServer) protected readonly applicationServer: ApplicationServer,
```

Comment:
We usually don't change constructor signatures, even when something is unused. Maybe mark the field as deprecated? `/** @deprecated since 1.6.0 */`